dlp-terraform/ecs/alb.tf

```diff
@@ -0,0 +1,63 @@
+# --- ALB ---
+resource "aws_security_group" "http" {
+  name_prefix = "http-sg-"
+  description = "Allow all HTTP/HTTPS traffic from public"
+  vpc_id      = aws_vpc.main.id
+
+  dynamic "ingress" {
+    for_each = [80, 443]
+    content {
+      protocol    = "tcp"
+      from_port   = ingress.value
+      to_port     = ingress.value
+      cidr_blocks = ["0.0.0.0/0"]
+    }
+  }
+
+  egress {
+    protocol    = "-1"
```

Comment:
outta curiosity, what does this mean?

Reply:
It allows outbound traffic on any protocol (tcp or udp)

Reply:
got it